pkg/pillar/cmd/domainmgr/domainmgr.go

```
@@ -2750,6 +2758,15 @@ func createCloudInitISO(config types.DomainConfig) (*types.DiskStatus, error) {
 	return ds, nil
 }
 
+//  decrypt the cipher text into plain text
+func cipherDecrypt(cipherInfo *types.CipherInfo,
+	cipherText []byte) (string, error) {
+	if !tpmmgr.IsTpmEnabled() || cipherInfo == nil || len(cipherText) == 0 {
+		return "", nil
```

Need to return an error if there is no TPM